lib/translator/translation.cc

```diff
@@ -121,10 +122,29 @@ BeginResponse Translation::Begin(Span<const uint8_t> scsi_cmd,
       pipeline_status_ = VerifyToNvme(scsi_cmd_no_op, nvme_cmds_[0]);
       nvme_cmd_count_ = 1;
       break;
+    case scsi::OpCode::kWrite6:
+      pipeline_status_ = Write6ToNvme(scsi_cmd, nvme_cmds_[0], allocations_[0],
+                                      nsid, kPageSize, kLbaSize);
+      nvme_cmd_count_ = 1;
+      break;
+    case scsi::OpCode::kWrite10:
+      pipeline_status_ = Write10ToNvme(scsi_cmd, nvme_cmds_[0], allocations_[0],
+                                       nsid, kPageSize, kLbaSize);
+      nvme_cmd_count_ = 1;
+      break;
+    case scsi::OpCode::kWrite12:
+      pipeline_status_ = Write12ToNvme(scsi_cmd, nvme_cmds_[0], allocations_[0],
+                                       nsid, kPageSize, kLbaSize);
+      nvme_cmd_count_ = 1;
+      break;
+    case scsi::OpCode::kWrite16:
+      pipeline_status_ = Write16ToNvme(scsi_cmd, nvme_cmds_[0], allocations_[0],
+                                       nsid, kPageSize, kLbaSize);
+      nvme_cmd_count_ = 1;
+      break;
     default:
       DebugLog("Bad OpCode: %#x", static_cast<uint8_t>(opc));
       pipeline_status_ = StatusCode::kFailure;
-      break;
```

Comment:
Let's leave this break statement here for consistency and robustness to future refactoring.

Reply:
sounds good. added it back it